sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2701,6 +2701,20 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val INSERT_OVERWRITE_TRASH_INTERVAL =
+    buildConf("spark.insertOverwrite.trash.interval")
```

Comment:
Maybe, you didn't clean up `insertOverwirte` code here.

Reply:
Done